<!--
マウス（タッチパッド）操作サンプルのlaunch

rosserialを使ってROSとメガローバーを接続してから使用してください。

起動コマンド：roslaunch megarover_samples mousectrl.launch
-->

<launch>
  <!-- マウスによる制御ノード -->
  <node pkg="mouse_teleop" type="mouse_teleop.py" name="mouse_teleop">
    <remap from="/mouse_vel" to="/rover_twist" />
  </node>
</launch>
